src/plugin/ipc/socket/socketconnection.cpp

```diff
@@ -95,7 +127,7 @@ void SocketConnection::restoreSocketOptions(vector<int>& fds)
       int ret = _real_setsockopt(fds[0], lvl->first, opt->first,
                                  opt->second.buffer(),
                                  opt->second.size());
-      JASSERT(ret == 0) (JASSERT_ERRNO) (fds[0])
+      JWARNING(ret == 0) (JASSERT_ERRNO) (fds[0])
```

Comment:
I assume the reason for this is that if the socket is a netlink, then we expect the call to `setsockopt()` to fail.
But then shouldn't we also add a `JASSERT` in the situation where we know that the socket is not associated with netlink?